enterprise/backend/src/metabase_enterprise/advanced_config/models/pulse_channel.clj

```diff
@@ -39,5 +39,4 @@
                                (pr-str domain)
                                (str/join ", " allowed-domains))
                           {:email           email
-                           :allowed-domains allowed-domains
-                           :status-code     403})))))))
```

Comment:
This leads to 
<img width="525" alt="Screen Shot 2021-09-23 at 7 07 05 pm" src="https://user-images.githubusercontent.com/8542534/134717563-8077d1c1-8bfd-4c7b-9e02-99531f0f48f3.png">